### PR TITLE
override _log_progress() in word2vec to track training loss

### PR DESCRIPTION
**Pull request by usul83**
I came across this suggestion from @villmow in this thread https://github.com/RaRe-Technologies/gensim/issues/1172 and thought I'd implement since @menshikh-iv thought it was a good idea and doesn't seem to have been done by anyone else.

First time I've contributed so appologies if not followed correct process etc.

### CODE REVIEW

**Comment by menshikh-iv** [comment]
Hi @usul83,

as I see, this is definition from https://github.com/RaRe-Technologies/gensim/blob/408a714e836484e914a7e3622b4f0f99a6d27ddd/gensim/models/base_any2vec.py#L1148, why this copy-paste needed (this method is inherited automatically)?

**Comment by usul83** [comment]
It should be a little different from the any2vec version, it adds: self.get_latest_training_loss() / example_count

**Comment by menshikh-iv** [comment]
@usul83 but in this case, `compute_loss` **must be** True, otherwise - this will not work, need to check this case.

**Comment by usul83** [comment]
It won't work, but it will give a 0 rather than raise an error (since self.running_training_loss = 0 at line 556 whether compute_loss true or not). Maybe could add a check and a warning for that situation?

**Comment by menshikh-iv** [comment]
No warning needed, simply check, if `compute_loss` is False - use parent method, otherwise - your.

**Comment by menshikh-iv** [comment]
this PR fully overlapped by #2135, for this reason, I close current.
@usul83 thanks for your work!
